### DISCUSSION TITLE
About usage of Enums

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.Native#41] [General]
:wave:

I see a "mix" regarding usage of enums; as example, some method require the enum explicitly:

https://github.com/dlemstra/Magick.Native/blob/d0a63d7f3852652c7c5c68c1950a00de015878f5/src/Magick.Native/MagickImage.h#L171

And some doesn't:

https://github.com/dlemstra/Magick.Native/blob/d0a63d7f3852652c7c5c68c1950a00de015878f5/src/Magick.Native/MagickImage.h#L433

Is it possible to standardize - one way or the other ? (I don't know the consequences, something like ABI break maybe?)

Regards.

**Comment by dlemstra** [comment]
It should be `const size_t` in all the exported symbols. And making that change should have no effect so I just pushed a patch to resolve that.

**Reply by Gounlaf** [reply]
Thanks!
